types/onoff.d.ts

```diff
@@ -0,0 +1,42 @@
+
+declare type Directions = "in" | "out";
+declare type Edges = "none" | "rising" | "falling" | "both";
+declare type GpioOptions = {
+    debounceTimeout: number,
+    activeLow: boolean,
+    reconfigureDirection: boolean,
+}
```

Comment:
If a `GpioOptions` object is passed as an argument when constructing a Gpio, the `debounceTimeout`, `activeLow` and `reconfigureDirection` options are all optional. I think this means it should be `debounceTimeout?`, `activeLow?` and `reconfigureDirection?`.

Reply:
Yes, indeed.

Talking about optional paramters: I have a minor "complaint" about how you handle callback parameters. Would you like me to open a seperate issue for that?

Reply:
Sure, feel free to open an issue.